src/postgres/tree/PostgresDatabaseTreeItem.ts

```diff
@@ -70,6 +70,8 @@ export class PostgresDatabaseTreeItem extends AzureParentTreeItem<ISubscriptionC
                 const host: string = nonNullProp(this.parent.server, 'fullyQualifiedDomainName');
                 const clientConfig: ClientConfig = { user: username, password, ssl, host, port: 5432, database: this.databaseName };
                 const accountConnection: Client = new Client(clientConfig);
+
+                await this.parent.checkAndConfigureFirewall();
```

Comment:
Okay well now that we merged the other PRs, this behavior doesn't match anymore. Can we make this more similar to "Enter server credentials"? Aka it doesn't prompt automatically and just shows a "GenericTreeItem" if I need to set it up